Makefile

```diff
@@ -61,11 +61,11 @@ test: bin/phpunit ## Run the test suite
 
 .PHONY: lint
 lint: bin/phpcs ## Run the linter on the PHP files
-	$(PHP) ./bin/phpcs . --standard=phpcs.xml --warning-severity=0 --extensions=php -p
+	$(PHP) ./bin/phpcs . -p -s
```

Comment:
@aledeg Does that look good to you?

Reply:
Now file extensions are defined in `phpcs.xml`
`-s` is to include the name of broken rules in the outputs